test/e2e/authorizer/serviceaccounts_test.go

```diff
@@ -162,8 +162,6 @@ func TestServiceAccounts(t *testing.T) {
 		}},
 	}
 	for i, ttc := range testCases {
-		i := i
```

Comment:
I remember reading some go release notes but cant find any about this. We don't need this anymore, right? Or what is the reasons to removing these?

Reply:
its a closure thing with variables... Still confused why we removing this?

Reply:
Loop variables are copies since Go 1.22, so creating scoped copies is not necessary anymore.

See also https://go.dev/blog/go1.22#language-changes